test/integration/xds_integration_test_base.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include "envoy/api/v2/discovery.pb.h"
+#include "envoy/grpc/status.h"
+#include "envoy/stats/scope.h"
+
+#include "common/config/resources.h"
+#include "common/protobuf/protobuf.h"
+#include "common/protobuf/utility.h"
+
+#include "test/integration/http_integration.h"
+#include "test/integration/utility.h"
+#include "test/mocks/server/mocks.h"
+#include "test/test_common/network_utility.h"
+#include "test/test_common/simulated_time_system.h"
+#include "test/test_common/utility.h"
+
+#include "gtest/gtest.h"
+
+using testing::AssertionFailure;
+using testing::AssertionResult;
+using testing::AssertionSuccess;
+using testing::IsSubstring;
+
+namespace Envoy {
+
+class XdsIntegrationTestBase : public HttpIntegrationTest {
```

Comment:
Refactor looks good 👍

Reply:
FWIW I'd actually be happy to move this into the http integration test, or just integration test  I'd like being able to test reloadable config be a thing which can easily be done in any test :-)

Reply:
I think this is legit, but I think until we start adding this to more tests, it's fine where @fredlas has placed it.